js/src/redux/selectors/results.js

```diff
@@ -23,6 +25,45 @@ export function getResultsForKeyword( state, keyword ) {
 	return get( seoResults, keyword, [] );
 }
 
+/**
+ * Gets the readability results for the focus keyword.
+ *
+ * @param {object} state The state.
+ *
+ * @returns {object} The readability results and score for the focus keyword.
```

Comment:
I think you mean 'overall score for the readability' instead of 'score for the focus keyword'